src/EnergyPlus/DataGlobalConstants.hh

```diff
@@ -129,6 +129,7 @@ namespace DataGlobalConstants {
         Condensate,
         OtherFuel1,
         OtherFuel2,
+        DistrictHeatingSteam,
```

Comment:
Does numbering make a difference here? Should this be moved next to `DistrictHeating` and `DistrictCooling`? 📌  need to do something about this and FuelType. Not in this branch.